src/components/Heading1.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Heading = styled.h1`
+  color: ${(props) => props.theme.headingBlue};
+`;
+
+const Heading1 = ({content}) => {
+    return (
```

Comment:
A small thing for me I really like to remove anything not in use so could just return without the curly brackets, but maybe makes sense to leave in case we are doing some logic in here soon etc. If we removed it then could be

```javascript
const Heading1 = ({content}) => 
        <Heading>
            {content}
        </Heading>
```

Another options is since content is being used as children can use the children prop that we get in react. 

```javascript
const Heading1 = ({children}) => 
        <Heading>
            {children}
        </Heading>


and then to use it we just wrap around our stuff

<Heading1>some header</Heading1>

```

Reply:
But yes let's merge and then iterate as we add more stuff :-)

Reply:
yeah I think we'll come back to it anyways